assets/python/ml/model.yaml

```diff
@@ -0,0 +1,239 @@
+  kernel: "linear"
+
+---
+
+name: LASSO
+category: regression
+provider: sklearn
+ensemble: False
+description: >
+  Workflow unit for a LASSO-regression model with Scikit-Learn. Model parameters derived from Scikit-Learn's Defaults. Alpha has been lowered from the default of 1.0, to 0.1.
+imports:
+  - sklearn.linear_model
+model_class: sklearn.linear_model.Lasso
+model_default_args:
+  alpha: 0.1
+  fit_intercept: True
+  normalize: False
+  precompute: False
+  tol: 1.e-4
+  positive: True
+  selection: "cyclic"
+
+---
+
+name: multilayer_perceptron
+category: regression
+provider: sklearn
+ensemble: False
+description: >
+  Workflow unit to train a simple feedforward neural network model on a regression problem using scikit-learn. In this template, we use the default values for hidden_layer_sizes, activation, solver, and learning rate. Other parameters are available (consult the sklearn docs), but in this case, we only include those relevant to the Adam optimizer. Sklearn Docs: Sklearn docs:http://scikit-learn.org/stable/modules/generated/sklearn.neural_network.MLPRegressor.html
+imports:
+  - sklearn.neural_network
+model_class: sklearn.neural_network.MLPRegressor
+model_default_args:
+  hidden_layer_sizes: (100,)
+  activation: "relu"
+  solver: "adam"
+  max_iter: 200
+  early_stopping: False
+  validation_fraction: 0.1
+
+---
+
+name: random_forest
+category: classification
+provider: sklearn
+ensemble: False
+description: >
+  Workflow unit for a random forest classification model with Scikit-Learn. Parameters derived from Scikit-Learn's defaults.
+imports:
+  - sklearn.ensemble
+model_class: sklearn.ensemble.RandomForestClassifier
+model_default_args:
+  n_estimators: 100
+  criterion: "gini"
+  max_depth: None
+  min_samples_split: 2
+  min_samples_leaf: 1
+  min_weight_fraction_leaf: 0.0
+  max_features: "auto"
+  max_leaf_nodes: None
+  min_impurity_decrease: 0.0
+  bootstrap: True
+  oob_score: False
+  verbose: 0
+  class_weight: None
+  ccp_alpha: 0.0
+  max_samples: None
+
+---
+
+name: random_forest
+category: regression
+provider: sklearn
+ensemble: False
+description: >
+  Workflow for a random forest regression model with Scikit-Learn. Parameters are derived from Scikit-Learn's defaults.
+imports:
+  - sklearn.ensemble
+model_class: sklearn.ensemble.RandomForestRegressor
+model_default_args:
+  n_estimators: 100
+  criterion: "mse"
+  max_depth: None
+  min_samples_split: 2
+  min_samples_leaf: 1
+  min_weight_fraction_leaf: 0.0
+  max_features: "auto"
+  max_leaf_nodes: None
+  min_impurity_decrease: 0.0
+  bootstrap: True
+  max_samples: None
+  oob_score: False
+  ccp_alpha: 0.0
+  verbose: 0
+
+---
+
+name: ridge
+category: regression
+provider: sklearn
+ensemble: false
+description: >
+  Workflow unit for a ridge regression model with Scikit-Learn. Alpha is taken from Scikit-Learn's default parameters.
+imports:
+  - sklearn.linear_model
+model_class: sklearn.linear_model.Ridge
+model_default_args:
+  alpha: 1.0
```

Comment:
There has to be some documentation explaining what's going on with the YAML config and auto-generated `pyi` code - let's add it to the README, pls.

Reply:
Added a readme to this directory